<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget {
background-color: rgb(215, 229, 255)
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Welcome">
     <widget class="QLabel" name="image1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/ADMIN/Pictures/Final_project/tải xuống.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Login">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
font-size: 14px
}</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Register">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
font-size: 14px
}</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Login">
     <widget class="QLabel" name="image2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/ADMIN/Pictures/Final_project/tải xuống.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 20px;
font-weight: bold;
}</string>
      </property>
      <property name="text">
       <string>Hello</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Sign into your Account</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Email_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 12px;
}</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Password_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 12px;
}</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Email_LineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
font-size: 12px
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password_LineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
font-size: 12px
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Login_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ForgotPass">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background: transparent;
border: none;
color: #rgb(123, 123, 123);
font-size: 12px;
text-align: right;
}

</string>
      </property>
      <property name="text">
       <string>Forgot your password?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>56</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background: transparent;
border: none;
color: #rgb(123, 123, 123);
font-size: 12px;
text-align: left;
}

</string>
      </property>
      <property name="text">
       <string>&lt; Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 12px;
}</string>
      </property>
      <property name="text">
       <string>Don't have an account ?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Register_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>370</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	text-decoration: underline;
	font: 8pt &quot;MS Shell Dlg 2&quot;;
background: transparent;
border: none;
color: #rgb(123, 123, 123);
font-size: 12px;
}

</string>
      </property>
      <property name="text">
       <string>Register now</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Register">
     <widget class="QLabel" name="image3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/ADMIN/Pictures/Final_project/tải xuống.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Back_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>56</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background: transparent;
border: none;
color: #rgb(123, 123, 123);
font-size: 12px;
text-align: left;
}

</string>
      </property>
      <property name="text">
       <string>&lt; Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password_LineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
font-size: 12px
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="UserName_LineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
font-size: 12px
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="Email_Label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 12px;
}</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserName_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 12px;
}</string>
      </property>
      <property name="text">
       <string>User Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Password_Label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 12px;
}</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Email_LineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
font-size: 12px
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Create New Account</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Register_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 12px;
}</string>
      </property>
      <property name="text">
       <string>Already have an account ?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Login_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>380</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	text-decoration: underline;
	font: 8pt &quot;MS Shell Dlg 2&quot;;
background: transparent;
border: none;
color: #rgb(123, 123, 123);
font-size: 12px;
}

</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 20px;
font-weight: bold;
}</string>
      </property>
      <property name="text">
       <string>Hello</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Reset_Password">
     <widget class="QLabel" name="NewPassword_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 12px;
}</string>
      </property>
      <property name="text">
       <string>New Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NewPassword_LineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
font-size: 12px
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="RePassword_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 12px;
}</string>
      </property>
      <property name="text">
       <string>Re - enter New Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RePassword_LineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
font-size: 12px
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="image4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/ADMIN/Pictures/Final_project/tải xuống.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Back_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>56</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background: transparent;
border: none;
color: #rgb(123, 123, 123);
font-size: 12px;
text-align: left;
}

</string>
      </property>
      <property name="text">
       <string>&lt; Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Reset Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Finish_btn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>302</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
